internal/server/errors.go

```diff
@@ -54,18 +58,7 @@ func sendAPIError(c *gin.Context, err error) {
 		resp.Message = authzError.Error()
 
 	case errors.As(err, &uniqueConstraintError):
-		resp.Code = http.StatusConflict
-		resp.Message = err.Error()
-		// remove the error trace from field error message
-		errMsg := err.Error()
-		errTrace := strings.Split(err.Error(), ":")
-		if len(errTrace) > 0 {
-			errMsg = errTrace[len(errTrace)-1]
-		}
```

Comment:
It's not obvious from the diff (since this code moved), but I changed how we accomplished this. I noticed there was a leading space in the error message, and I believe it was coming from here.

Instead of trimming the error, I use the message from `uniqueConstraintError.Error()` instead of `err.Error()`. The `uniqueConstraintError.Error()` doesn't have the "trace" part of the error because those parts are removed by `errors.As`, which calls `Unwrap` to remove them.

I updated the test case to show it still works the same way (and fixes the leading space in the error message).